Charts/Classes/Components/ChartYAxis.swift

```diff
@@ -254,7 +254,7 @@ public class ChartYAxis: ChartAxisBase
             return ""
         }
         
-        return (valueFormatter ?? _defaultValueFormatter).string(from: entries[index])!
+        return (valueFormatter ?? _defaultValueFormatter).string(for: entries[index])!
```

Comment:
I am not sure if we need to use for (default imp will crash) or use NSNumber. Have you test it yet?